lib/plaid/connection.rb

```diff
@@ -9,22 +9,29 @@ class << self
       def post(path, options = {})
         uri = build_uri(path)
         options.merge!(client_id: Plaid.customer_id, secret: Plaid.secret)
-        res = Net::HTTP.post_form(uri,options)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == 'https')
```

Comment:
Yes, as per https://github.com/plaid/plaid-ruby#configuring-plaid

<pre>
Plaid.config do |p|
  p.customer_id = 'Plaid provided customer ID here'
  p.secret = 'Plaid provided secret key here'
  p.environment_location = 'URL for the development or production environment'
  # i.e. 'https://tartan.plaid.com/' for development, or
  # 'https://api.plaid.com/' for production
end
</pre>

We either use 'https://tartan.plaid.com/' or 'https://api.plaid.com/'. So both the cases it would be https only and ssl will be set true.

> i think Plaid always uses SSL
> I think this http vs https logic could be safely replaced for strict http in current master code too.

Yes, we should strictly use https i.e :use_ssl => true for plaid requests.

Thanks,
Nitesh